docs/guides/dns-over-https.md

```diff
@@ -2,20 +2,20 @@
 
 DNS-Over-HTTPS is a protocol for performing DNS lookups via the same protocol you use to browse the web securely: **HTTPS**.
 
-With standard DNS, requests are sent in plain-text, with no method to detect tampering or misbehaviour. This means that not only can a malicous actor look at all the DNS requests you are making (and therefore what websites you are visiting), they can also tamper with the response and redirect your device to resources in their control (such as a fake login page for internet banking).
+With standard DNS, requests are sent in plain-text, with no method to detect tampering or misbehavior. This means that not only can a malicious actor look at all the DNS requests you are making (and therefore what websites you are visiting), they can also tamper with the response and redirect your device to resources in their control (such as a fake login page for internet banking).
```

Comment:
>With standard DNS, requests are sent in plain-text, with no method to detect tampering or misbehavior.

>they can also tamper with the response and redirect your device to resources in their control (such as a fake login page for internet banking).

This is factually incorrect. DNSSEC specifically protects from this.

Reply:
That's something not directly related to my patch though :P

Reply:
Point taken.